namnder/idrottsnamnden/body.md

```diff
@@ -19,7 +19,7 @@ De aktiviteter vi främst kommer ägna oss åt är sporterna som ingår i THS-M
 
 Om du är sugen på att syssla med någon annan idrott så hör gärna av dig, det mesta går att ordna!
 
-Är du intresserad av att delta i någon av dessa aktiviteter så gå med i [facebook-gruppen](https://www.facebook.com/groups/datafotboll/) eller hör av dig direkt till Sektionsidrottsledaren via __idrott(at)d.kth.se__. Du kan också skicka önskemål om aktiviteter du vill genomföra till denna adress. En annan informationskanal idrottsnämnden ämnar använda är nyheterna på sektionens hemsida.
+Är du intresserad av att delta i någon av dessa aktiviteter så gå med i [facebook-gruppen](https://www.facebook.com/groups/datafotboll/) eller hör av dig direkt till Sektionsidrottsledaren via [idrott@datasektionen.se](mailto:idrott@datasektionen.se). Du kan också skicka önskemål om aktiviteter du vill genomföra till denna adress. En annan informationskanal idrottsnämnden ämnar använda är nyheterna på sektionens hemsida.
```

Comment:
ohh this is my bad. https://www.facebook.com/groups/datakthsport changed the URL of the group...

Reply:
I can fix it later.

Reply:
```suggestion
Är du intresserad av att delta i någon av dessa aktiviteter så gå med i [facebook-gruppen](https://www.facebook.com/groups/datakthsport) eller hör av dig direkt till Sektionsidrottsledaren via [idrott@datasektionen.se](mailto:idrott@datasektionen.se). Du kan också skicka önskemål om aktiviteter du vill genomföra till denna adress. En annan informationskanal idrottsnämnden ämnar använda är nyheterna på sektionens hemsida.
```